Platform/obc/hal/obc_hal.c

```diff
@@ -83,3 +83,11 @@ size_t HAL_hk_report(uint8_t sid, void *output) {
       return 0;
   }
 }
+
+/* This is a test to see if the data is sent back successfully.
+ * Once decided on how, the real hal connected to equipment handler
+ * will be implemented */
+void HAL_comm_getTemp(uint32_t *sensor_temperature){
+    *sensor_temperature = 18;
+    //HAL_get_temperature(sensor_temperature);
```

Comment:
@ArashYzd one note: When you start adding more of the actual functionality, lets put this stuff in `obc/hal/comms_hal.c&.h` so we keep things seperate.

Reply:
Sure! Noted! That was the kind of feedback I was looking for.